src/binwalk/magic/firmware

```diff
@@ -271,7 +271,7 @@
 #
 # Motorola S-Records, from Gerd Truschinski <gt@freebsd.first.gmd.de>
 # Improved by Martin Sundhaug <martinsundhaug@gmail.com>
-0   regex       (S[0-35-9]([0-9A-F]{4})([0-9A-F]{2})+\n)+       Motorola S-Record{many}; binary data in text format, record type:
```

Comment:
Why the removal of the {many} tag? It prevents binwalk from reporting multiple results when a S-record spans multiple blocks (binwalk reads in and scans files in blocks, rather than doing the whole file all at once which would be memory intensive for large files). 

Same question for the Intel Hex signature below.